python/importer/main.py

```diff
@@ -1,66 +1,94 @@
 """HFP Analytics data importer"""
 import azure.functions as func
-from azure.storage.blob import BlobServiceClient, BlobBlock
+from azure.storage.blob import BlobServiceClient
 from io import StringIO
 import os
 import csv
+import logging
 import zstandard
 from datetime import datetime, timedelta
 import psycopg2 as psycopg
-from common.utils import get_conn_params, get_logger
+from common.logger_util import PostgresDBHandler
+from common.utils import get_conn_params
 import common.constants as constants
 from .run_analysis import main as run_analysis
 
 # TODO: import other event types as well when needed.
 event_types_to_import = ['DOC', 'DOO']
 
-def main(dataImporter: func.TimerRequest):
-    logger = get_logger()
+def main(importer: func.TimerRequest, context: func.Context):
+    # Taken from: https://stackoverflow.com/a/879937/4282381
+    class AnalyticsLoggingFilter(logging.Filter):
```

Comment:
I think this is very OK for now - or, instead of a class we could also use a [factory function](https://rmcomplexity.com/article/2020/12/01/introduction-to-python-logging.html#custom-filter-example), but that's rather a matter of taste so don't worry about it too much. Anyway I think this class is worth moving under `common.logger_util` from where we can import it. This way `main()` remains a bit tidier (still with the logging boilerplate though), and this `'main'`-based filter could also be useful in possible future routines run similarly as `main`.